lib/maths/analytics/CBoostedTreeHyperparameters.cc

```diff
@@ -480,13 +483,6 @@ bool CBoostedTreeHyperparameters::selectNext(const TMeanVarAccumulator& testLoss
         std::copy(m_HyperparameterSamples[m_CurrentRound].begin(),
                   m_HyperparameterSamples[m_CurrentRound].end(), parameters.data());
         parameters = minBoundary + parameters.cwiseProduct(maxBoundary - minBoundary);
-        if (m_CurrentRound == 0) {
```

Comment:
Yes, because we now have estimates of the variance in the loss values for all the earlier rounds. We could look at early exiting during fine tune on any iteration. I was going to explore this in separate change.